app/grandchallenge/annotations/models.py

```diff
@@ -14,12 +19,73 @@ class AbstractAnnotationModel(UUIDModel):
     """
 
     grader = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
-    # Override inherited 'created' attribute
+    # Override inherited 'created' attribute to allow setting of value
     created = models.DateTimeField(default=timezone.now)
 
     class Meta:
         abstract = True
 
+    def save(self, *args, **kwargs):
+        """ Override save method to enable setting of permissions for retina users """
+        created = False
+        if self._state.adding:
+            created = True
+
+        super(AbstractAnnotationModel, self).save(*args, **kwargs)
+
+        if not created:
+            return
+
+        if (
+            self.grader.groups.filter(
+                name=settings.RETINA_GRADERS_GROUP_NAME
+            ).exists()
+            or self.grader.groups.filter(
+                name=settings.RETINA_ADMINS_GROUP_NAME
+            ).exists()
+        ):
+            model_name = self.__class__.__name__.lower()
```

Comment:
Again, this should come from `_meta`

Reply:
Done in 3ce431a984d771bda25d6afd42fbf573f887e140